io.sourceforge.chart_geany.chart-geany.yml

```diff
@@ -0,0 +1,57 @@
+app-id: io.sourceforge.chart_geany.chart-geany
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp # otherwise webengine is not available
+base-version: '5.15-22.08'
+command: ChartGeany
+desktop-file-name-suffix: '' # used to create development version
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
can't we tighten filesystem permissions? This uses Qt so it should work with portal. or at least xdg-documents.

Reply:
so you would like to remove it completely?

Reply:
tighter permissions is always better.`xdg-documents` would be a good option if it doesn't support portals.